### DISCUSSION TITLE
Prevent Escape of Certain Characters in Markdown

### DISCUSSION THREAD

**Discussion by thekief** [jgm/pandoc#9568] [General]
I have a Typst temnplate that is filled with data from a markdown document. Due to the nature of the setup, I have the need to call typst functions in certain places of the markdown document.

I have the issue that Typst function calls/variables accesses are escaped, due to being prefixed with a `#`. Currently, I am able to fix that with a small post-processing step but that is not a viable generic approach. Does anybody know of an extension of flag that would make it possible to prevent certain escapes?

**Comment by jgm** [comment]
I'm unclear about what you're doing, but if you're asking how to leave Typst function calls unescaped when converting from Markdown, the answer is to use the `raw_attribute` syntax:
```
This will `#emph[be passed through literally into typst]`{=typst}
```

**Reply by thekief** [reply]
Thank you for your fast answer. That solved it it :grin:
